cli/readInputs.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+// This function allows for increased functionality of the Vault helper tool
+// Reads one line of user input at a time
+func readInput() {
+	fmt.Print("# Enter command or press q to quit: ")
```

Comment:
Needless text duplication; if possible, remove this line and move line 31 immediately underneath line 16:
```python
for scanner.Scan() {
    fmt.Print("# Enter command or enter q to quit: ")
    result := scanner.Text()
```

Reply:
I tried that initially, but since the for loop waits for input, it doesn't actually execute until a command is read in. This is what happens:
```bash
we
# Enter command or enter q to quit: Wrong command executed or wrong number of arguments. Use ./cli -help for more info
```
I can move this to a string variable for clarity however.